src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Circle extends Figure {
```

Comment:
The Circle class should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.